pyproject.toml

```diff
@@ -25,6 +25,7 @@ testing = ["pytest"]
 
 [tool.poetry.group.dev.dependencies]
 tbump = "^6.11.0"
+pytest = "^8.3.4"
```

Comment:
This is already in the `[tool.poetry.extras]` section above and can be installed via `poetry install -E testing`